boards/itsybitsy_m0/board_config.h

```diff
@@ -0,0 +1,20 @@
+#ifndef BOARD_CONFIG_H
+#define BOARD_CONFIG_H
+
+#define __SAMD21G18A__ 1
```

Comment:
You can remove `#define __SAMD21G18A__ 1` because it's automatically done by the `CHIP_VARIANT = SAMD21G18A` in the `board.mk`  file. This is done by line 26 in the Makefile:

`-D__$(CHIP_VARIANT)__ \`